utils/desktop_bridge.js

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+let callback;
+
+window.desktopBridge = {};
+
+export function connect(callbackFunction) {
+    // store provided callback for later use
+    callback = callbackFunction;
+
+    // connect bridge methods
+    window.desktopBridge.updateUserActivityStatus = updateUserActivityStatus;
+}
+
+const updateUserActivityStatus = (userIsActive) => {
```

Comment:
In order to facilitate an easier transition in the future should this API change, can we pass an object here instead ala `({userIsActive})`? Then the desktop app could add new parameters in the future as needed without having to worry about the parameter order to this method.

Reply:
Redid the bridge to mimick the existing functionality of Electron's ipcMain and ipcRenderer modules already used on the Desktop side of this feature. An arbitrary number of properties are now supported.